promotion/replication/replication_iskhakov_et_al_2016.ipynb

```diff
@@ -0,0 +1,1338 @@
+{
```

Comment:
<p>Instead of Loading all the data and results from the auxiliary file. Could you create a pickle, txt or json file where you store the processed data. The folder with the raw files could be confusing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/ruspy/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>